lib/pure/os.nim

```diff
@@ -88,7 +88,7 @@ proc existsFile*(filename: string): bool {.rtl, extern: "nos$1",
     return stat(filename, res) >= 0'i32 and S_ISREG(res.st_mode)
 
 proc existsDir*(dir: string): bool {.rtl, extern: "nos$1", tags: [ReadDirEffect].} =
-  ## Returns true iff the directory `dir` exists. If `dir` is a file, false
+  ## Returns true iff the directory `dir` exists (via a symlink is ok). If `dir` is a file, false
```

Comment:
I think the better term for this is "follows symlinks", to be consistent with

https://github.com/nim-lang/Nim/blob/269b957e2968345a4c9a5037ab97ab726135a921/lib/pure/os.nim#L319-L322

Reply:
done